addons/web/static/lib/owl/owl.js

```diff
@@ -1547,7 +1547,7 @@
                     let textNode;
                     while ((textNode = node.previousSibling) !== prevElem) {
                         if (textNode.nodeValue.trim().length) {
-                            throw new Error("text is not allowed between branching directives");
+                            //throw new Error("text is not allowed between branching directives");

```

Comment:
Add link to related OWL issue: https://github.com/odoo/owl/issues/636

Reply:
ok